protobuf/flow/access/access.proto

```diff
@@ -160,6 +162,15 @@ message GetAccountResponse {
   entities.Account account = 1;
 }
 
+message GetAccountAtBlockHeightRequest {
+  bytes address = 1;
+  uint64 block_height = 2;
+}
+
+message GetAccountAtBlockHeightResponse {
+  entities.Account account = 1;
+}
```

Comment:
Just like with the `GetBlock` methods, all of the `GetAccount` methods can share a common `AccountResponse`